Probesys/bileto
# tec: Refactor the answer form

## Related issue(s)



https://github.com/Probesys/bileto/issues/551

## How to test manually



1. Use the answer form
2. Verify everything works as expected

## Checklist
